src/bot.py

```diff
@@ -123,23 +120,16 @@ def tweet_loop(self, api):
                 else:
                     api.update_status(self.generate_batch())
                     logging.info("Tweeted out a new flower bed!")
-                    logging.info("The next tweet is scheduled to be made in {} seconds".format(cooldown))
+                    logging.info("The next tweet is scheduled to be made in {} minutes".format(cooldown))
                     time.sleep(cooldown)
-                    raise tweepy.TweepError("error")
-            except Exception as ex:
-                exception = type(ex).__name__
-                
-                if exception == 'RateLimitError':
-                    logging.critical("Tweeting failed due to ratelimit. Waiting {} more seconds.".format(cooldown))
-                    time.sleep(cooldown)
-                else:
-                    logging.critical("TweepError not RateLimitError.")
-                    
+            except tweepy.RateLimitError:
+                logging.critical("Tweeting failed due to ratelimit. Waiting {} more minutes.".format(cooldown))
+                time.sleep(cooldown)
+

```

Comment:
There are two handlers for tweepy.RateLimitError

Reply:
I was experimenting with the code when I mistakenly removed those lines. I will try and keep un-related code untouched. Cheers!